test/test_sample.py

```diff
@@ -321,25 +322,32 @@ def test_multi_proc(self):
                 data=logistic_data,
                 chains=4,
                 parallel_chains=1,
-                show_progress=False,
+                show_console=True,
             )
         log.check_present(
-            ('cmdstanpy', 'INFO', 'finish chain 1'),
-            ('cmdstanpy', 'INFO', 'start chain 2'),
+            ('cmdstanpy', 'INFO', 'Chain [1] done processing'),
+            ('cmdstanpy', 'INFO', 'Chain [2] start processing'),
         )
+
+    # pylint: disable=no-self-use
+    def test_multi_proc_2(self):
+        logistic_stan = os.path.join(DATAFILES_PATH, 'logistic.stan')
+        logistic_model = CmdStanModel(stan_file=logistic_stan)
```

Comment:
Are we going to add a test which uses the new logic? We don't even need to compile with `STAN_THREADS=True`, as it should just run the chains sequentially if we don't (though I see no issue with passing that cpp option in the tests)

Reply:
yes

Reply:
added test for 2.28